allennlp/modules/token_embedders/embedding.py

```diff
@@ -140,6 +149,25 @@ def forward(self, inputs):  # pylint: disable=arguments-differ
             embedded = projection(embedded)
         return embedded
 
+    @overrides
+    def extend_vocab(self, extended_vocab: Vocabulary, vocab_namespace: Optional[str] = None):
+        if not vocab_namespace:
+            vocab_namespace = getattr(self, "_vocab_namespace", None)
```

Comment:
Can't you just do `self._vocab_namespace`?  This is still backwards compatible, because an old model will still use the current constructor, which sets `self._vocab_namespace = None`.

Reply:
Ohh right! This would still be backward compatible, will fix. 👍

Reply:
This also means you can just do `vocab_namespace = vocab_namespace or self._vocab_namespace`